### PR TITLE
New release of TimeTrackingEditor and Added BigBoard

### PR DESCRIPTION
**Pull request by stinnux**

### CODE REVIEW

File: plugins.json [hunk 1]
```diff
@@ -83,6 +83,18 @@
...
+        "title": "Bigboard",
+        "version": "1.0.2",
+        "author": "Stinnux",
+        "license": "MIT",
+        "description": "A Kanboard that can display multiple projects.",
+        "homepage": "https://github.com/stinnux/kanboard-bigboard",
+        "readme": "https://raw.githubusercontent.com/stinnux/kanboard-bigboard/master/README.md",
+        "download": "https://github.com/stinnux/kanboard-bigboard/releases/download/1.0.2/Bigboard-1.0.2.zip",
+        "remote_install": true,
+        "compatible_version": ">=1.1.22"
```
Comment (fguillot) [inline plugins.json]: There is a typo here, version 1.1.22 doesn't exists.

**Review by fguillot — CHANGES_REQUESTED** [review]

File: plugins.json (hunk 1 above)
Comment (stinnux) [reply plugins.json]: You are right, tyipo fixed.

File: plugins.json [hunk 2]
```diff
@@ -83,6 +83,18 @@
...
+        "title": "Bigboard",
+        "version": "1.0.2",
+        "author": "Stinnux",
+        "license": "MIT",
+        "description": "A Kanboard that can display multiple projects.",
+        "homepage": "https://github.com/stinnux/kanboard-bigboard",
+        "readme": "https://raw.githubusercontent.com/stinnux/kanboard-bigboard/master/README.md",
+        "download": "https://github.com/stinnux/kanboard-bigboard/releases/download/1.0.2/Bigboard-1.0.2.zip",
+        "remote_install": true,
+        "compatible_version": ">=1.2.22"
```
Comment (fguillot) [inline plugins.json]: 1.2.22 doesn't exists yet. The latest version is 1.2.3, see https://github.com/kanboard/kanboard/releases

**Review by fguillot — CHANGES_REQUESTED** [review]

**Comment by stinnux** [comment]
I hope this time i managed to do it correct.

**Pull request merged**